src/App.js

```diff
@@ -42,7 +43,7 @@ query {
   howIsAPersonReferredCollection{items{
     referredInfo
   }},
-  vissionOnServicesPageCollection{
+  visionOnServicesPageCollection{
     items{
       vission
```

Comment:
Do we want to update this vission too?

Reply:
The key stayed the same (I wasn't able to edit it without needing to redo the contentful model)